FlowCrypt/src/test/java/com/flowcrypt/email/security/pgp/Utils.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors:
+ *   Ivan Pizhenko
+ */
+
+package com.flowcrypt.email.security.pgp
+
+import org.bouncycastle.openpgp.PGPPublicKeyRing
+import org.bouncycastle.openpgp.PGPSecretKeyRing
+import org.pgpainless.PGPainless
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+
+object Utils {
```

Comment:
we already have a place for such code. Please look at `com.flowcrypt.email.util.TestGeneralUtil` and move code there.

Reply:
Unfortunately it gives build error when I move to that class. Note that `TestGeneralUtil`  is under `androidTest` and these tests are under `test`.

Reply:
ah... sorry. So we can leave it as-is